pocs/tests/test_constraints.py

```diff
@@ -197,3 +197,30 @@ def test_moon_avoidance(observer):
 
     assert veto1 is False and veto2 is False
     assert score2 > score1
+
+def test_horizon(observer):
+    h = Horizon()
+
+    time = Time('2016-08-13 10:00:00')
+
+    observation1 = Observation(Field('HD189733', '20h00m43.7135s +22d42m39.0645s'))  # HD189733
+    observation2 = Observation(Field('Hat-P-16', '00h38m17.59s +42d27m47.2s'))  # Hat-P-16
+
+    veto1, score1 = h.get_score(time, observer, observation1)
+    veto2, score2 = h.get_score(time, observer, observation2)
+
+    assert veto1 is False and veto2 is False
+    assert score2 > score1 #Are these scores still relevant? How can I test scores for my class as its either 100 or 0?
```

Comment:
I would just test one observation at at time and see if `score1 > 0` or something. Depends on what you are actually trying to test. Let me know if that doesn't seem to make sense.